packages/react-composites/src/composites/ChatComposite/FileDownloadCards.tsx

```diff
@@ -39,7 +42,21 @@ export interface FileDownloadCards {
 export const FileDownloadCards = (props: FileDownloadCards): JSX.Element => {
   const truncateLength = 15;
   const { userId, message } = props;
+  // @conditional-compile-remove(file-sharing)
```

Comment:
Why do we need to conditionally remove anything in this component?

Reply:
Done